pandas/core/indexes/range.py

```diff
@@ -380,8 +380,15 @@ def get_loc(self, key, method=None, tolerance=None):
 
     @Appender(_index_shared_docs["get_indexer"])
     def get_indexer(self, target, method=None, limit=None, tolerance=None):
-        if not (method is None and tolerance is None and is_list_like(target)):
-            return super().get_indexer(target, method=method, tolerance=tolerance)
+        if not (
+            method is None
+            and tolerance is None
+            and limit is None
```

Comment:
does this need to be part of the condition? the test could be parameterised on limit keyword.

Reply:
i think it can be in condition as others?

Reply:
looking at the implementation of `RangeIndex.get_indexer`, it does not support limit, so yes, this looks correct as is, it needs to be in the condition and call the method from the parent class.